docs/DeveloperGuide.md

```diff
@@ -245,73 +245,60 @@ The following activity diagram summarizes what happens when a user executes a ne
 
-### \[Proposed\] Data archiving
-
-_{Explain here how the data archiving feature will be implemented}_
-
-
-
 ### \[Implemented\] Status feature
 
 #### Implementation
 
-The implemented status label is facilitated by `Status`. It extends `AddressBook` with a Status, tied to each person. Additionally, it implements the following operations:
-
-* `Status#FUNCTIONNAME()` — DESCRIPTION.
-
-These operations are exposed in the `Model` interface as `Model#___()`, `Model#___()` and `Model#___()` respectively.
-
-Given below is an example usage scenario and how the undo/redo mechanism behaves at each step.
-
-Step 1. 
-
-![NAME PLACEHOLDER]()
-
-Step 2. 
-
-![NAME PLACEHOLDER]()
+The implemented status label is facilitated by `Status` attribute. This label is an additional attribute for each person within the application
+and is implemented as a separate file within the `Person` Package.
 
-Step 3. 
-
-![NAME PLACEHOLDER]()
-
-Step 4. 
-
-![NAME PLACEHOLDER]()
-
-The following sequence diagram shows how the undo operation works:
+#### Design considerations:
 
-![NAME PLACEHOLDER]()
+**Aspect: Abstracting `Status` attribute**
 
-Step 5. 
+* **Alternative 1 (current choice):** Abstracted class.
+  * Pros: 
+    * Higher Level of abstraction
+    * Changes can be made easily from this class
+  * Cons: 
+    * Adding files to the currently already large amount of files within
+    * Difficulty navigating through folders to find specific files
 
-![NAME PLACEHOLDER]()
+* **Alternative 2:** Attribute placed within `Person` class.
+  * Pros: 
+    * Single file where changes can be made
+  * Cons: 
+    * Lesser level of abstraction, changes made have to be constantly changed throughout the file
 
-Step 6. 
+### \[Implemented\] Find By Status feature
 
-![NAME PLACEHOLDER]()
+#### Implementation
 
-The following activity diagram summarizes what happens when a user executes a new command:
+The implemented find by status mechanism is facilitated by `findstatus` command. It extends `UDT` with a Find By Status, allowing users to find persons by their current COVID-19 statuses.
 
-<img src="IMAGE PATH" width="250" />
+Classes added for this feature: 
+* `StatusContainsKeywordsPredicate`
+* `FindStatusCommand`
+* `FindStatusCommandParser`
 
 #### Design considerations:
 
-**Aspect: PLACEHOLDER:**
-
-* **Alternative 1 (current choice):** PLACEHOLDER.
-  * Pros: PLACEHOLDER.
-  * Cons: PLACEHOLDER.
-
-* **Alternative 2:** PLACEHOLDER.
-  * Pros: PLACEHOLDER.
-  * Cons: PLACEHOLDER.
-
-### \[Implemented\] Find By Status feature
+**Aspect: Abstracting into different classes**
 
-#### Implementation
+* **Alternative 1 (current choice):** Abstracted classes.
+    * Pros:
+        * Higher Level of abstraction
+        * Changes can be made easily from this class
+        * Different classes serve different lower-level purposes and are organised into separate packages (e.g. `FindStatusCommandParser` belongs to the `parser` package)
```

Comment:
This sentence has a good structure. Moreover, it describes the pro and provides an example to support the statement! Perhaps the Abstracting "Status" section can follow this structure!